test/c_api_tests.cpp

```diff
@@ -4,6 +4,8 @@
 #include <models/model.h>
 #include <iostream>
 #include <ort_genai.h>
+#include "../src/span.h"
```

Comment:
The idea in this test file is that we only use the publicly available header file and not rely on the internal span functionality.
Could we avoid using this include?

Reply:
This is a header only feature that substitutes for standard header for non-C++20 environments.
In C++20 it reduces to a standard header.
It creates no binary dependency, and therefore it is OK to use it.